package.json

```diff
@@ -27,27 +27,26 @@
   },
   "dependencies": {
     "@sentry/node": "^5.27.1",
-    "bluebird": "^3.1.1",
-    "escape-string-regexp": "^2.0.0",
-    "extend": "^2.0.0",
-    "he": "^1.1.1",
-    "matrix-org-irc": "1.0.0-alpha4",
-    "js-yaml": "^3.14.0",
+    "bluebird": "^3.7.2",
+    "escape-string-regexp": "^4.0.0",
+    "extend": "^3.0.2",
```

Comment:
3.0.0 introduced a global "strict" directive, which shouldn't break anything for us.
https://github.com/justmoon/node-extend/blob/main/CHANGELOG.md#300--2015-07-01